sys-cluster/teleport/teleport-2.2.0.ebuild

```diff
@@ -27,8 +27,15 @@ DEPEND="
 	>=dev-lang/go-1.8.3"
```

Comment:
Besides, it's your choice but normally you don't have to patch old versions, just the newest.